pkg/server/container_create_unix.go

```diff
@@ -172,6 +172,9 @@ func (c *criService) containerSpec(id string, sandboxID string, sandboxPid uint3
 		specOpts = append(specOpts, oci.WithPrivileged)
 		if !ociRuntime.PrivilegedWithoutHostDevices {
 			specOpts = append(specOpts, oci.WithHostDevices, oci.WithAllDevicesAllowed)
+		} else {
+			// add requested devices by the config as host devices are not automatically added
+			specOpts = append(specOpts, customopts.WithDevices(c.os, config), customopts.WithCapabilities(securityContext))
```

Comment:
I believe we are doing what is expected here by allowing a user to configure CRI to not pass host devices to privileged containers. The default behavior is to pass them through. Thoughts?
```
      # privileged_without_host_devices allows overloading the default behaviour of passing host
      # devices through to privileged containers. This is useful when using a runtime where it does
      # not make sense to pass host devices to the container when privileged. Defaults to false -
      # i.e pass host devices through to privileged containers.
      privileged_without_host_devices = false
```
will run the bucket to show the test result hopefully we catch the change as an error :-)

Reply:
Without means not.. IOW the default for the flag is false... which negates the without.. causing it to pass the host devices through.  Set it to true and it withholds (without) passing the host devices through.